src/server-helpers.ts

```diff
@@ -81,9 +79,7 @@ export const createUploadFailureHandler =
     const failedFile = Object.values(req?.files || [])
       .flat()
       .find(({ truncated }) => truncated);
-    if (failedFile) {
-      return next(error);
-    }
+    if (failedFile) return next(error);
     next();
```

Comment:
can use ternary here?

Reply:
technically should be fine, but no such pattern in docs

https://github.com/pillarjs/router/blob/0d2cefc7dc978cd34db6baad7d3a71ae2888eda3/lib/route.js#L122-L131

Reply:
keep it